server/src/main/java/com/objectcomputing/checkins/services/member_skill/MemberSkillController.java

```diff
@@ -56,25 +58,16 @@ public MemberSkillController(MemberSkillServices memberSkillServices,
                 .body(error);
     }
 
-    @Error(exception = MemberSkillBadArgException.class)
-    public HttpResponse<?> handleBadArgs(HttpRequest<?> request, MemberSkillBadArgException e) {
-        JsonError error = new JsonError(e.getMessage())
-                .link(Link.SELF, Link.of(request.getUri()));
-
-        return HttpResponse.<JsonError>badRequest()
-                .body(error);
-    }
-
-    @Error(exception = MemberSkillAlreadyExistsException.class)
-    public HttpResponse<?> handleAlreadyExists(HttpRequest<?> request, MemberSkillAlreadyExistsException e) {
+    @Error(exception = AlreadyExistsException.class)
+    public HttpResponse<?> handleAlreadyExists(HttpRequest<?> request, AlreadyExistsException e) {
```

Comment:
Don't think we need this. Should be handled by the global handler AlreadyExistsHandler in the exceptions/handlers folder